spec/models/host_spec.rb

```diff
@@ -462,6 +462,24 @@ def assert_remote_credentials_validated
     end
   end
 
+  describe "#clustered" do
```

Comment:
Are these specs related?

On that note should there be specs on the new relation or specs removed for the new relation?